test/extensions/filters/http/wasm/test_data/queue_cpp.cc

```diff
@@ -43,7 +47,15 @@ void ExampleRootContext::onQueueReady(uint32_t token) {
     logInfo("onQueueReady");
   }
   std::unique_ptr<WasmData> data;
-  if (WasmResult::Ok == dequeueSharedQueue(token, &data)) {
+  if (dequeueSharedQueue(9999999 /* bad token */, &data) != WasmResult::NotFound) {
+    // Should not be called.
+    logWarn("onQueueReady bad token found");
```

Comment:
Nit: I'd prefer you could turn those into positive tests, i.e.
```
if (dequeueSharedQueue(9999999 /* bad token */, &data) == WasmResult::NotFound) {
  logWarn("onQueueReady bad token not found");
}
```
so that we could verify that those things are actually happening.

(same in other tests)